pom.xml

```diff
@@ -72,6 +72,12 @@
       <artifactId>jackson-core</artifactId>
       <version>${jackson.version.core}</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <version>3.0.2</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
You might drop this now since its not module friendly (though replacements are a big mess). iirc scala had bugs where it crashed if annotations are missing from the classpath

Reply:
thanks @ben-manes, I'll see about dropping this dependency - I did test this locally in Scala (the jackson-module-scala plugin) and it seemed ok but it might be best to drop the annotations altogether

Reply:
It may be fixed in scala, the last time I tried to dig up the bug I found these references,
- https://www.scala-lang.org/old/node/10575
- https://github.com/scala/bug/issues/7751
- https://web.archive.org/web/20170110134053/https://medium.com/mibexsoftware/class-file-is-broken-scala-compiler-error-cbe33e5cd805